packages/@scope/jsii-calc-lib/package.json

```diff
@@ -34,7 +34,8 @@
     "@scope/jsii-calc-base": "^1.1.0"
   },
   "peerDependencies": {
-    "@scope/jsii-calc-base": "^1.1.0"
+    "@scope/jsii-calc-base": "^1.1.0",
+    "@scope/jsii-calc-base-of-base": "^1.1.0"
```

Comment:
Why is this now required? Weird.

Reply:
Always has been, jsii emits a warning for it, which has always been ignored. The package exposes a type from jsii-calc-base-of-base, so it needs to declare a peered on it.